### PR TITLE
Enter/Leave messages, status and activities

### PR DESCRIPTION
**Pull request by tiagojpdias**
- Add MAIN_CHANNEL_ID as an environment variable so we can define the main channel to send enter/leave messages.
- Set activity/status according to enter/leave.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```
Comment (quetzyg) [inline .vscode/launch.json]: Is this shit even legal?

File: app.js [hunk 2]
```diff
@@ -1,27 +1,65 @@
...
-const client = new Discord.Client();
+const client = new Client();
+
+const files = [
+  'https://i.kym-cdn.com/photos/images/original/001/213/604/4c6.png',
+];
+
+const messages = {
+  enter: 'bom dia a todos menos a um',
+  exit: 'XAU AI _BORDAS_',
```
Comment (quetzyg) [inline app.js:13]: :heart:

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (quetzyg) [inline .vscode/launch.json:1]: Shouldn't this file be ignored?

**Review by quetzyg — CHANGES_REQUESTED** [review]
Left a few comments

File: .vscode/launch.json (hunk 1 above)
Comment (tiagojpdias) [reply .vscode/launch.json]: Will remove that comment.

File: .vscode/launch.json (hunk 3 above)
Comment (tiagojpdias) [reply .vscode/launch.json:1]: Good question.
Added it to allow consistency among devs which use vscode and want to debug in the same circumstances.

Don't know if should add or not. What do you think?

Comment (zambinij) [reply .vscode/launch.json:1]: It's fine by me.

**Review by zambinij — APPROVED** [review]

File: .vscode/launch.json (hunk 3 above)
Comment (quetzyg) [reply .vscode/launch.json:1]: Leave it, then.

**Review by quetzyg — APPROVED** [review]

**Pull request merged**